plugins/org.eclipse.fordiac.ide.export.forte_ng/src/org/eclipse/fordiac/ide/export/forte_ng/ForteFBTemplate.xtend

```diff
@@ -383,13 +383,13 @@ abstract class ForteFBTemplate<T extends FBType> extends ForteLibraryElementTemp
 		«ENDIF»
 	'''
 	
-	//todo escape new line character
 	def protected generateSetInitialValuesDefinition(Iterable<VarDeclaration> variables) '''
 		«IF(containsNonRetainedVariable(variables))»
 			void «className»::setInitialValues() {
 				«generateVariableDefaultAssignment(variables.filter[!isRetainedVariable(it)])»
 			}
 		«ENDIF»	
+		
```

Comment:
Please move the empty line inside the `ENDIF` (where it was initially), so that it is only added when a `setInitialValues` is present.